### ISSUE TITLE
Comments on "Install the Dynamics 365 app [...]"

### ISSUE THREAD

**Issue by fmarcellino** [MicrosoftDocs/power-platform#1894]
Hello,

I'd like to propose fixing the typo below - "Intall" -> "Install", and also add a comment on that option because there are customers that can't see that in PPAC. What are the conditions to see it?

_Intall the Dynamics 365 app for Teams for users in the organization. Lets people in your org see business data while working in Teams._

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: b83794a9-896d-a4a8-85f3-858c9f3591b6
* Version Independent ID: b83794a9-896d-a4a8-85f3-858c9f3591b6
* Content: [Manage collaboration settings - Power Platform](https://learn.microsoft.com/en-au/power-platform/admin/settings-collaboration)
* Content Source: [power-platform/admin/settings-collaboration.md](https://github.com/MicrosoftDocs/power-platform/blob/main/power-platform/admin/settings-collaboration.md)
* Service: **power-platform**
* Sub-service: **admin**
* GitHub Login: @fikaradz
* Microsoft Alias: **fikaradz**

**Comment by fikaradz** [comment]
Yes,  let's fix the typo.  The option is rolling out right now, it's available in station 1, and will be in all stations by end of May.
